Lib/idlelib/editor.py

```diff
@@ -791,6 +799,8 @@ def ResetFont(self):
         # Called from configdialog.py
 
         self.text['font'] = idleConf.GetFont(self.root, 'main','EditorWindow')
+        if self.codecontext is not None:
```

Comment:
It might be better (later) to change context font first, since effect of font change on text depends on effect of font change on context, but not vice versa.  See review comment.  Possible code:
```
        newfont = idleConf.GetFont(self.root, 'main','EditorWindow')
        if self.codecontext is not None:
           self.codecontext.update_font(newfont)  # change sig
        self.text['font'] = newfont
        if self.linenumber is not None: ...
```

Reply:
I was OK with leaving this for a later PR, but it may actually make changes even snappier.  In reading ResetFont, I noticed that it is ResetColorizer, above this, also called from configdialog and somewhere else, that deletes and restores colors.  We should try reconfiguring tags instead on another issue.  Names should sometime be lower-cased.